Cataclysm/Druid.lua

```diff
@@ -752,7 +781,7 @@ spec:RegisterStateExpr("clip_roar", function()
     
     -- Calculate when roar would end if casted now
     -- Calculate roar duration since aura.savage_roar.duration gives wrong values
```

Comment:
Not sure why, but the `aura.savage_roar.duration` gives wrong values

Reply:
works for me:
![grafik](https://github.com/Hekili/hekili/assets/126069971/ae02f963-5758-4e15-92ab-4899920765e5)
![grafik](https://github.com/Hekili/hekili/assets/126069971/3607049b-113c-480e-bcdd-7b8e58da0676)

Did you try dumping the value using `/dump Hekili.State.aura.savage_roar.duration` cause it would show you the calced duration after your last recommended action (and probably a CP spender):
![grafik](https://github.com/Hekili/hekili/assets/126069971/64ab749a-5095-46c7-88a4-5e46274bfcb7)

Reply:
It seems the problem is not only with Savage Roar, but I do see a lot of inconsistency in the state variables, especially with 5 and 0 combo points. I have been investigating how combo points are calculated and I still don't fully understand it, nor do I see a bug. I know the WoW API is called every cycle, and it is stored as one of the resources and the update is forced when the combo points change. I think it's worth looking into it, maybe for another PR.

Macro:
```
/run print("CP from wow: "..GetComboPoints( "player", "target" ))
/dump Hekili.State.aura.savage_roar.duration
/dump Hekili.State.combo_points.current
```
![image](https://github.com/Hekili/hekili/assets/35671015/51194e7e-7a5d-4352-ab90-8d4b9057a0e3)

Reply:
You don't know that you have a sane, fresh addon state when you dump values from `Hekili.State` in chat.  In fact, you almost certainly will never get a fresh state that way.  The addon resets and then starts manipulating it based on its predictions and next steps.

You can `/run Hekili.State.reset()` at the start of the same macro to force a fresh state.